hardware/raspberrypi/boot_diagnostics.md

```diff
@@ -0,0 +1,28 @@
+## Boot Diagnostics on the Raspberry Pi 4
+
+The bootloader on the Raspberry Pi 4 is stored in EEPROM, and as well as providing boot services, is also capable of providing diagnostic information on an attached HDMI display. To display this information, remove the SD card from the Raspberry Pi 4, and reboot. A diagnostic display similar to below should appear on the attached display.
```

Comment:
The text here about "remove the SD card from the Raspberry Pi 4, and reboot" implies that the user should remove the SD card while Raspbian is running, which probably isn't what we should be recommending?! :wink: 
Perhaps "power down the Raspberry Pi 4, remove the SD card, and then reapply power" ?